applications/tari_base_node/src/builder.rs

```diff
@@ -217,18 +242,27 @@ pub fn configure_and_initialize_node(
                 id.clone(),
                 peers,
                 &config.peer_db_path,
+                &config.data_dir,
```

Comment:
There's already `[wallet.wallet_file]` specified in the`config/tari_config_sample.toml` as well as in `default_config()` in `configuration.rs`.

`data_dir` is used for the Peer database.

What's currently missing is Bringing that info along into the `GlobalConfig` struct. (I think long term we'll want to break this up into separate configs for Node, Miner, Wallet etc, but we can use GlobalConfig for now)